vpc.tf

```diff
@@ -1,11 +1,11 @@
 variable "region" {
-  default     = "us-east-2"
+  default     = "eu-central-1"
   description = "AWS region"
 }
 
 provider "aws" {
   version = ">= 2.28.1"
-  region  = "us-east-2"
+  region  = "eu-central-1"
```

Comment:
why changed region?

Reply:
I think that Europe is better for our current needs. because Frankfurt is closer than Ohio. I changed the region and in README in the description